tests/sentry/web/frontend/test_vercel_extension_configuration.py

```diff
@@ -58,8 +58,16 @@ def test_logged_in_one_org(self):
         req_params = parse_qs(mock_request.body)
         assert req_params["code"] == ["my-code"]
 
+        app_install = SentryAppInstallationForProvider.objects.get(
+            organization=self.org, provider="vercel"
+        )
+
         # Goes straight to Vercel OAuth
         assert resp.status_code == 302
+        assert (
+            resp.url
+            == f"http://testserver/settings/{self.org.slug}/integrations/vercel/{app_install.id}/?next=https%3A%2F%2Fexample.com"
```

Comment:
nit: Is it possible to test these query params as dict somehow? I picture something like 
```python
params = {next: example.com}
params_str = create_param_string(params)
assert resp.url = f".../integrations/vercel/{params_str}
```
or better yet, somehow reversing it

```python
params_str = get_params_from_url(resp.url)
params = {next: example.com}
for key, val in decode(params_str).items():
  assert params[key] = value
```
that way if the order of the keys changes on the url, he test doesnt flake.